src/components/JoinUs/ViewOpenPositions.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Padding } from 'styled-components-spacing'
+import { Grid, Row, Col } from '../grid'
+import StyledLink from '../styledLink'
+import { H2 } from '../Typography'
+
+const ViewPositions = ({ text }) => (
+  <Grid>
+    <Padding
+      top={{ smallPhone: 4, smallTablet: 0 }}
+      bottom={{ smallPhone: 3.5, desktop: 5 }}
```

Comment:
Hmm, not really. What I was saying above was about spacing between the navbar & this component's title (i.e. top padding)

This comment is a separate and very minor tweak to the bottom padding. What you've got is right 3 out of 4 breakpoints (mobile, small tablet & desktop). But to make this conform to Zeplin, you're just missing `tablet: 5`.

Does that make sense?

Reply:
the `smallTablet: 0.5` was to add to the natural distance caused by the top menu height, in order to create the distance that we see on zepling — from the text container to the top of the window. Let's see what Antonas and Tom decide regarding this?

Reply:
I don't understand. `smallTablet: 0.5`? Where's that?

Maybe we should just jump on a call tomorrow?

Reply:
probably! ping me when your available

Reply:
Was just checking, the padding on small tablet is 54px === 3.5, right?

![image](https://user-images.githubusercontent.com/3236388/51535134-7ff44a80-1e3f-11e9-85dd-cff07a110943.png)

Reply:
Sorry! Screenshot above is from large tablet, not small tablet.

My bad :sweat:

Reply:
@jsms90 I've talked with antonas and he told me to follow the padding rules for the 'About Us' page (https://app.zeplin.io/project/5b4e666f0d4230a16dffd93a/screen/5c112c2ac5dd4c508a0eaadc). And based on that, the paddings for this components are correct. The visual discrepancy we see on the top spacing, when comparing with the designs, it's because our header has wrong paddings coded. I've created a ticket addressing that and have put it on our 'Ready for Engineering' list - https://trello.com/c/FoKYTdV9